xml/bp_troubleshooting_cephadm.xml

```diff
@@ -90,7 +90,7 @@
 <screen>
 &prompt.cephuser;ceph orch ps --daemon-type prometheus
 NAME               HOST    STATUS         REFRESHED  AGE  VERSION  IMAGE NAME                                                   IMAGE ID      CONTAINER ID
-prometheus.master  master  running (65m)  7m ago     2h   2.27.1   registry.suse.com/ses/7/prometheus/prometheus-server:2.27.1  e77db6e75e78  7b11062150ab
+prometheus.master  master  running (65m)  7m ago     2h   2.32.1   registry.suse.com/ses/7/prometheus/prometheus-server:2.27.1  e77db6e75e78  7b11062150ab
```

Comment:
It seems only the first occurrence of 2.27.1 was replaced here. The latter one would also need to be replaced.